tests/Behat/ContentLockContext.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_editorial\Behat;
+
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+
+/**
+ * Defines step definitions for testing the Content Lock feature.
+ */
+class ContentLockContext extends RawDrupalContext {
+
+  /**
+   * Install the content_lock component.
+   *
+   * @BeforeScenario @content_lock
+   */
+  public function enableContentLock(): void {
+    // Enable the Content Lock feature.
+    \Drupal::service('module_installer')->install(['oe_editorial_content_lock']);
+  }
+
+  /**
+   * Uninstall the content_lock component.
+   *
+   * @AfterScenario @content_lock
+   */
+  public function disableContentLock(): void {
+    $modules = [
+      'oe_editorial_content_lock',
+      'content_lock',
+    ];
+    // Remove the Content Lock feature and its dependencies.
+    foreach ($modules as $module) {
+      \Drupal::service('module_installer')->uninstall([$module]);
+    }
+  }
+
+}
```

Comment:
I would remove this entirely, we enable it via the task runner, we should use this method only when we are testing optional sub-modules.

Reply:
But the content_lock module doesn't get enabled on the task runner as of now, it is an optional module. If we enable it by default in the task runner I think it will start screwing with all the other tests

Reply:
You are right! Sorry about that, somehow I missed it! :+1: